mmv1/templates/terraform/examples/firewall_policy_rule.tf.erb

```diff
@@ -1,6 +1,6 @@
 resource "google_network_security_address_group" "basic_global_networksecurity_address_group" {
```

Comment:
could you address the merge conflicts that are found in this file?

Reply:
Sure, just did!